### ISSUE TITLE
[Models listing] Ensure aria-hidden elements are not focusable nor contain focusable elements

### ISSUE THREAD

**Issue by YuriiDavydiak** [epam/ai-dial-admin-frontend#36]
### EPAM AI DIAL Admin version

0.4.0

### How to reproduce

Tested with Accessibility Insights for Web Google Chrome extension, using Fast run (level A)

### Actual result

![Image](https://github.com/user-attachments/assets/6a6b5bff-ea00-460f-97fa-5c705e69f029)

### Expected result

no errors are displayed 

### Confidential information

- [x] I confirm that do not share any confidential information

**Comment by YuriiDavydiak** [comment]
fixed

<img width="1913" height="1025" alt="Image" src="https://github.com/user-attachments/assets/687eaad3-fba3-4ba1-a24d-ff572901f7f0" />
